style.css

```diff
@@ -0,0 +1,479 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+html {
+  font-size: 15px;
+  font-family: 'montserratregular', san-serif;
+}
+
+body {
+  margin: 0 auto;
+  position: relative;
+  min-width: 320px;
+  max-width: 1440px;  
+}
+
+h1,
+h2 {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  font-family: 'montserratsemibold', san-serif;
+  text-decoration: none;
+  color: black;
+}
+
+button {
+  border: none;
+  background: none;
+  cursor: pointer;
+  outline: none;
+  display: flex;
+  align-items: center;
+  padding: 0 40px;
+  color: black;
+  font-size: 13px;
+  font-family: 'montserratbold', san-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__pic {
+  margin: 10px;
+  width: 166px;
+  height: 51px;
+}
+
+.menu {
+  padding: 0;
+}
+
+.header__aside {
+  display: flex;
+  margin: 30px 0 0 0;
+}
+
+.header__menu-link {
+  display: inline-block;
+  margin: 0 20px;
+  color: black;
+  transition: color 0.1s liner;
+  position: relative;
+}
+
+.header__menu-link::after {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 2px;	
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)